include/plc_iface.h

```diff
@@ -18,6 +18,9 @@
  NOTE: this file is not preproccessed, so no # macros (#define, #include, etc.)
  */
 
+#ifndef _PLC_IFACE_H_
```

Comment:
now this will not work with python CFFI, this is the reasoning for the comment above. please revert and refrain from using any preprocessor macros in this file.

Reply:
Ok . desestimate this commit